Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest-problems/ContestProblems.tsx

```diff
@@ -0,0 +1,83 @@
+import { useEffect } from 'react';
+
+import { IProblemType } from '../../../common/types';
+import useTheme from '../../../hooks/use-theme';
+import { setSelectedContestDetailsProblem } from '../../../redux/features/contestsSlice';
+import { useAppDispatch, useAppSelector } from '../../../redux/store';
+
+import styles from './ContestProblems.module.scss';
+
+interface IContestProblemsProps {
+    problems: Array<IProblemType>;
+    onContestProblemChange: () => void;
+    totalParticipantsCount?: number;
+    sumMyPoints?: number;
+    sumTotalPoints: number;
+}
+
+const ContestProblems = (props: IContestProblemsProps) => {
+    const { problems, onContestProblemChange, totalParticipantsCount, sumMyPoints = 0, sumTotalPoints } = props;
+
+    const dispatch = useAppDispatch();
+    const { themeColors, getColorClassName } = useTheme();
+    const { selectedContestDetailsProblem } = useAppSelector((state) => state.contests);
+
+    const backgroundColorClassName = getColorClassName(themeColors.baseColor200);
+    const darkBackgroundClassName = getColorClassName(themeColors.baseColor500);
+
+    useEffect(() => {
+        if (problems.length > 0) {
+            dispatch(setSelectedContestDetailsProblem({ selectedProblem: problems[0] }));
+        }
+    }, [ problems, problems.length, dispatch ]);
+
+    const onProblemClick = (problem: IProblemType) => {
+        onContestProblemChange();
+        dispatch(setSelectedContestDetailsProblem({ selectedProblem: problem }));
+    };
+
+    return (
+        <div>
+            <div className={styles.wrapper}>
+                <div className={`${styles.problemsWrapper} ${backgroundColorClassName}`}>
+                    <div className={styles.problemsHeader} style={{ borderBottom: `1px solid ${themeColors.textColor}` }}>
+                        <div>Tasks</div>
+                        <div>Points</div>
+                    </div>
+                    {problems.map((problem) => {
+                        const isActive = selectedContestDetailsProblem?.id === problem.id;
+                        return (
+                            <div
+                              className={`${styles.problem} ${isActive
+                                  ? styles.activeProblem
+                                  : ''}`}
+                              style={{ borderBottom: `1px solid ${themeColors.textColor}` }}
```

Comment:
Remove inline styling

Reply:
I dont have an idea how to group that border without in line styling, Im open for suggestions about that one.